packages/api/src/types.ts

```diff
@@ -28,15 +29,14 @@ import {
 } from '@cennznet/types/types';
 import {DeriveCustom} from '@plugnet/api-derive';
 import ApiBase from '@plugnet/api/base';
+import {ApiOptions as ApiOptionsBase, SignerOptions as SignerOptionsBase, UnsubscribePromise} from '@plugnet/api/types';
+
 import {
-    ApiOptions as ApiOptionsBase,
-    SignerOptions as SignerOptionsBase,
     SubmitableResultResult,
     SubmitableResultSubscription,
     SubmittableExtrinsic as SubmittableExtrinsicBase,
     SubmittableResultImpl,
-    UnsubscribePromise,
-} from '@plugnet/api/types';
+} from '@cennznet/api/submittable/types';
```

Comment:
should use relative path here, since what you import here is within same package.